app/Domains/Contact/ManageLifeEvents/Services/EditLifeEvent.php

```diff
@@ -0,0 +1,150 @@
+<?php
+
+namespace App\Domains\Contact\ManageLifeEvents\Services;
+
+use App\Interfaces\ServiceInterface;
+use App\Models\Contact;
+use App\Models\LifeEvent;
+use App\Models\LifeEventType;
+use App\Models\TimelineEvent;
+use App\Services\BaseService;
+use Carbon\Carbon;
+use Illuminate\Support\Collection;
+
+class EditLifeEvent extends BaseService implements ServiceInterface
```

Comment:
All the services in the app that edit something are called `UpdateXXX`, not `EditXXX`. Therefore, this service should be named `UpdateLifeEvent`.